app/src/main/java/com/wpi/audiojournal/screen/ProgramCategoriesScreen.kt

```diff
@@ -0,0 +1,107 @@
+package com.wpi.audiojournal.screen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import com.wpi.audiojournal.MenuItem
+import com.wpi.audiojournal.R
+
+@Composable
+fun ProgramCategoriesScreen(menuItems: List<MenuItem>, title: String, navController: NavController) {
+    //BACK BUTTON
+
+    Button(
+        modifier = Modifier
+            .padding(
+                horizontal = 5.dp,
+                vertical = 5.dp
+            )
+            //.fillMaxSize()
+            .size(width = 60.dp, height = 50.dp),
+        shape = RoundedCornerShape(30),
+        onClick = { navController.navigate("home") }
```

Comment:
So while "navController.navigate("home")" works in this instance, as a back button it probably makes more sense to use "navController.popBackStack()", since in Android when you move to a new screen it pops it to the top of the stack, so pressing back would remove the screen on the top of the stack and return the next highest screen on the stack, which would be the screen that a user was on before the current one.

Reply:
Yeah that feels like a more intuitive implementation for a back button.